docs/sources/administration/home_dashboard.md

```diff
@@ -0,0 +1,36 @@
++++
```

Comment:
Please change the file name to change-home-dashboard.md. Always match the topic title as closely as possible.